cmd/git-init/main.go

```diff
@@ -25,19 +25,27 @@ import (
 )
 
 var (
-	url                    = flag.String("url", "", "The url of the Git repository to initialize.")
-	revision               = flag.String("revision", "", "The Git revision to make the repository HEAD")
-	path                   = flag.String("path", "", "Path of directory under which git repository will be copied")
-	terminationMessagePath = flag.String("terminationMessagePath", "/dev/termination-log", "Location of file containing termination message")
-	submodules             = flag.Bool("submodules", true, "Initialize and fetch git submodules")
+	fetchSpec              git.FetchSpec
+	submodules             bool
```

Comment:
Is there a reason `submodules bool` isn't a field in `FetchSpec`?

Reply:
yep – it’s not an input to `git.Fetch`

Reply:
Ah right, thanks for explaining that.